### ISSUE TITLE
MVs should validate gc grace seconds on the tables involved

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla#2483]
https://issues.apache.org/jira/browse/CASSANDRA-9917

**Comment by duarten** [comment]
We already do: https://github.com/scylladb/scylla/blob/master/cql3/statements/create_view_statement.cc#L186

**Comment by slivne** [comment]
in Cassnarda they also do that in LOGGED_BATCH_STATEMENTS

**Comment by duarten** [comment]
Right, but that's an issue for `batch_statement`, unrelated to MVs, no?

**Comment by slivne** [comment]
well it may be more critical for the logged batches of mv statements as well - yet I changed the title and marked it as compatibility as we did not sort out the logged batches for mvs
